05week/spaceTravelToMars.js

```diff
@@ -9,6 +9,36 @@ let jobTypes = {
   programmer: 'Any Ship!'
 };
 
+class CrewMember {
+  constructor(name, job, specialSkill){
+    this.name = name;
+    this.job = job;
+    this.specialSkill = specialSkill;
+    this.ship = null;
+  }
+  enterShip(ship)  {
+    this.ship = ship;
+    ship.crew.push(this);
+  }
+}
+
+class Ship {
+  constructor(name, type, ability, ship){
+    this.name = name;
+    this.type = type;
+    this.ability = ability
+    this.crew = [];
+  }
+  missionStatement(){
+    if(this.crew.length >= 1){
+      return this.ability;
+    } else {
+      return "Can't perform a mission yet.";
+    }
+  }
+}
+
+
 // Your code here
```

Comment:
You should make the a instance of the CrewMember and the Ship.
and have that person enter the ship.